src/storage/txn/scheduler.rs

```diff
@@ -369,12 +369,17 @@ fn process_read(cid: u64, mut cmd: Command, ch: SendCh<Msg>, snapshot: Box<Snaps
         Command::Gc { ref ctx, safe_point, ref mut scan_key, .. } => {
             let mut reader =
                 MvccReader::new(snapshot.as_ref(), Some(ScanMode::Forward), true, None);
+            let start_key = scan_key.clone();
```

Comment:
YES. Range start gc with scan_key=none, and scan_key will be used as start key(not none) in later steps.